core/tst/software/aws/toolkits/core/utils/DelegateSdkConsumers.kt

```diff
@@ -19,27 +20,34 @@ import kotlin.reflect.full.isSubclassOf
  *
  * This will handle "simple" methods that they generate as well such as ListBuckets that takes 0 arguments.
  */
-class DelegateSdkConsumers : Answer<Any> {
+class DelegateSdkConsumers(private val sdkClass: Class<*>) : Answer<Any> {
     override fun answer(invocation: InvocationOnMock): Any? {
         val method = invocation.method
-        return if (method.isDefault &&
-            method?.parameters?.getOrNull(0)?.type?.kotlin?.isSubclassOf(SdkRequest::class) != true &&
-            method.name !in setOf("utilities", "waiter") // Never call the real methods they use to get to their helper classes since it always throws
```

Comment:
`utilities` was deleted from this logic, is that OK?

Reply:
`software.aws.toolkits.jetbrains.services.s3.S3VirtualBucketTest#getUrl` still passes and it was added for that test so I don't think so

Reply:
Cool